### DISCUSSION TITLE
How to do self-service/self-registration

### DISCUSSION THREAD

**Discussion by mango009** [zitadel/zitadel#4306] [Q&A] [answered]
I'm trying to use `Zitadel` as the **IAM** for the IT infrastructure in our organization. All the examples shows how to do login for existing users. But how do we do self-service so that when a user does not have an account they can use self-service to create a new account? 

`Ory Kratos` describes it's approach in their [document](https://www.ory.sh/docs/kratos/self-service) quite clearly. Is there an equivalent for `Zitadel`?

**Comment by mango009** [comment]
It seems that the self-service is buried in the UI. The document seems bit thin in explaining self-service mechanism

**Reply by mffap** [reply]
We've added a section about self-service capabilities to our docs: https://docs.zitadel.com/docs/concepts/features/selfservice

**Comment by mffap** [comment, marked as answer]
Hey @mango009, yes certainly a point where the docs can be improved. Currently self-register is activated by default. You can enable/disable the Register functionality under "Loin Behavior and Security" under the settings either for the whole instance or for individual organizations. With that enabled users can self-register with ZITADEL.
